docs/_docs/1_1_spark_quick_start_guide.md

```diff
@@ -300,6 +300,268 @@ spark.
   show(100, false)
 ```
 
```

Comment:
@pengzhiwei2018  hello, now spark sql  support compaction 、cleaner、clustering ？

Reply:
Yes, I am planning to support this in the next PRs.